lsstswBuild.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -e
```

Comment:
*sigh*. Obviously I am not asking you to comment on every case individually. You clearly thought the change was important to do and you clearly think it's a justifiable improvement. Your commit message was not really clear why you made the change. At the very least improve the commit message. I assume you think I am asking because I don't know how write bash scripts but I don't think one line somewhere to explain yourself will hurt. If it's not "every script" that has the `set` wrapping how do you justify wrapping the ones you do wrap. What changed recently that caused you to make this modification? You must have been motivated by some error condition? Put that justification in the commit message.